src/main/java/in/twizmwaz/cardinal/module/modules/titleRespawn/TitleRespawnBuilder.java

```diff
@@ -0,0 +1,32 @@
+package in.twizmwaz.cardinal.module.modules.titleRespawn;
+
+import in.twizmwaz.cardinal.match.Match;
+import in.twizmwaz.cardinal.module.BuilderData;
+import in.twizmwaz.cardinal.module.ModuleBuilder;
+import in.twizmwaz.cardinal.module.ModuleCollection;
+import in.twizmwaz.cardinal.module.ModuleLoadTime;
+import in.twizmwaz.cardinal.util.StringUtils;
+import org.jdom2.Element;
+
+@BuilderData(load = ModuleLoadTime.NORMAL)
+public class TitleRespawnBuilder implements ModuleBuilder {
+
+    @Override
+    public ModuleCollection load(Match match) {
+        Element head = match.getDocument().getRootElement().getChild("respawn");
```

Comment:
@dentmaged but I need to use

```java
if (head == null) {
    return new ModuleCollection(new TitleRespawn(2, false, false, false, false));
}
```

because if the XML doesn't has a `<respawn />` tag the `head` will be `null`, so this https://github.com/TheMolkaPL/CardinalPGM/commit/8a4f0aa67cf24d4b276f8772480356a4c89604a5#diff-8989480d1f8f58581249e0dcac07d95bR23 will be a `NullPointerException`.